src/Microsoft.AspNetCore.Razor.Evolution/Legacy/CSharpTokenizer.cs

```diff
@@ -168,6 +168,181 @@ protected override StateResult Dispatch()
+                    case CSharpSymbolType.LeftBrace:
+                        return "{";
+                    case CSharpSymbolType.Or:
+                        return "|";
+                    case CSharpSymbolType.RightBrace:
+                        return "}";
+                    case CSharpSymbolType.Tilde:
+                        return "~";
+                    case CSharpSymbolType.Plus:
+                        return "+";
+                    case CSharpSymbolType.LessThan:
+                        return "<";
+                    case CSharpSymbolType.Assign:
+                        return "=";
+                    case CSharpSymbolType.GreaterThan:
+                        return ">";
+                    case CSharpSymbolType.Hash:
+                        return "#";
+                    case CSharpSymbolType.Transition:
+                        return "@";
+
+                }
+            }
+            else if (symbolLength == 2)
+            {
+                switch (type)
+                {
+                    case CSharpSymbolType.NewLine:
+                        return "\r\n";
+                    case CSharpSymbolType.Arrow:
+                        return "->";
+                    case CSharpSymbolType.Decrement:
+                        return "--";
+                    case CSharpSymbolType.MinusAssign:
+                        return "-=";
+                    case CSharpSymbolType.NotEqual:
+                        return "!=";
+                    case CSharpSymbolType.ModuloAssign:
+                        return "%=";
+                    case CSharpSymbolType.AndAssign:
+                        return "&=";
+                    case CSharpSymbolType.DoubleAnd:
+                        return "&&";
+                    case CSharpSymbolType.MultiplyAssign:
+                        return "*=";
+                    case CSharpSymbolType.DivideAssign:
+                        return "/=";
+                    case CSharpSymbolType.DoubleColon:
+                        return "::";
+                    case CSharpSymbolType.NullCoalesce:
+                        return "??";
+                    case CSharpSymbolType.XorAssign:
+                        return "^=";
+                    case CSharpSymbolType.OrAssign:
+                        return "|=";
+                    case CSharpSymbolType.DoubleOr:
+                        return "||";
+                    case CSharpSymbolType.PlusAssign:
+                        return "+=";
+                    case CSharpSymbolType.Increment:
+                        return "++";
+                    case CSharpSymbolType.LessThanEqual:
+                        return "<=";
+                    case CSharpSymbolType.LeftShift:
+                        return "<<";
+                    case CSharpSymbolType.Equals:
+                        return "==";
+                    case CSharpSymbolType.GreaterThanEqual:
+                        if (Buffer[0] == '=')
```

Comment:
[Yes](https://github.com/aspnet/Razor/blob/dev/src/Microsoft.AspNetCore.Razor.Evolution/Legacy/CSharpTokenizer.cs#L111).
I don't know why it is the same token type but...